ktor-http/ktor-http-cio/jvm/src/io/ktor/server/cio/backend/ServerPipeline.kt

```diff
@@ -155,7 +156,9 @@ public fun CoroutineScope.startServerConnectionPipeline(
                     )
                 } catch (cause: Throwable) {
                     requestBody.close(cause)
-                    throw cause
+                    response.writePacket(BadRequestPacket.copy())
+                    response.close()
+                    break
```

Comment:
What will happen with the requests that already running? Since we don't crash here anymore, they don't get cancelled, right?